src/scripts/main.js

```diff
@@ -1,11 +1,27 @@
-'use strict';
+"use strict";
 
 function waitFor(element, eventName) {
-// write your code here
+  const resolver = (resolve) => {
+    element.addEventListener(eventName, () => {
+      const message =
+        `It was ${eventName}` +
+        ` on the element: ${element.nodeName}, id: ${element.id}.` +
+        ` (For example: It was click on the element: INPUT, id: login.`;
+
+      printMessage(message);
+      resolve();
```

Comment:
```suggestion
      resolve(message);
```
or just write the message directly as an argument of `resolve`, without creating a separate variable
the `printMessage` will be used as a callback of `then` method in tests, so the text will be passed there without directly calling it here